python/src/aac/plugins/gen_plugin/templates/third_party/setup.py.jinja2

```diff
@@ -24,12 +24,13 @@ test_dependencies = [
 setup(
     version="0.0.1",
     name="{{plugin.name}}",
+    package_data={"": ["*.aac", "*.jinja2", "*.yaml"]},
```

Comment:
This is untested but, according to [this](https://docs.python.org/3/distutils/setupscript.html#installing-package-data), I think this may be what you want:

```suggestion
    packages=["{{plugin.name}}"],
    package_dir={"{{plugin.name}}": "src/{{plugin.name}}"},
    package_data={"{{plugin.name}}": ["*.aac", "*.jinja2", "*.yaml"]},
```

If that doesn't work, you may need to change the references to `{{plugin.name}}` with `{{plugin.implementation_name}}` but it should be pretty easy to test each of them to see what works.

Reply:
I have this in my local but it didnt get pushed up for some reason

Reply:
sorry I thought they were in local but was missing some of my previous implementation. Should be pushing up here soon.

Reply:
alright changes should pushed through...I think during my local testing I found that the `plugin.implementation_name` seemed to play better than the `plugin.name` due to the file structure was named from the former.

Reply:
The `packages=find_packages(where=".", exlude="tests")` is what got it going.